src/utils/types.ts

```diff
@@ -332,10 +342,20 @@ interface SectionDetails {
 
 interface ReviewProgress {
   totalReviewable: number
-  doneConform: number
-  doneNonConform: number
-  doneNewReviewable: number
-  totalNewReviewable: number
+  totalActive: number
+  doneConform?: number
+  doneNonConform?: number
+  doneNewReviewable?: number
+  totalNewReviewable?: number
+  totalConform?: number
+  totalNonConform?: number
+  totalPendingReview?: number
+  doneAgreeConform?: number
+  doneAgreeNonConform?: number
+  doneDisagree?: number
+  doneActiveDisagree?: number
+  doneActiveAgreeConform?: number
+  doneActiveAgreeNonConform?: number
```

Comment:
I had a longer look now and the names of these fields are very descriptive, so that's good. I think I understood most of it now. Just couldn't figure out exactly the meaning of the following ones:
    totalActive
    doneActiveDisagree
    doneActiveAgreeConform
    doneActiveAgreeNonConform

Reply:
53e1d3efc5d0567881630a2606d04bd7e3122bf1

I've added a comment in types file in reviewAndConsolidationProgress, does that makes sense ?